### ISSUE TITLE
reboot_timeout option is not recognised

### ISSUE THREAD

**Issue by BentoumiTech** [ansible/ansible#47503]
##### SUMMARY
reboot_timeout option is not recognised

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME
- reboot module

##### ANSIBLE VERSION
```paste below
ansible 2.7.0
  config file = /Users/ordinateur/Projects/ansible/ansible.cfg
  configured module search path = ['/Users/ordinateur/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/lib/python3.7/site-packages/ansible
  executable location = /usr/local/bin/ansible
  python version = 3.7.0 (default, Jul 23 2018, 20:22:55) [Clang 9.1.0 (clang-902.0.39.2)]
```

##### CONFIGURATION

```paste below
ANSIBLE_PIPELINING(/Users/ordinateur/Projects/ansible/ansible.cfg) = True
ANSIBLE_SSH_ARGS(/Users/ordinateur/Projects/ansible/ansible.cfg) = -o ServerAliveInterval=30 -o ControlMaster=auto -o ControlPersist=60s
ANSIBLE_SSH_CONTROL_PATH(/Users/ordinateur/Projects/ansible/ansible.cfg) = /tmp/ansible-ssh-%%h-%%p-%%r
DEFAULT_TIMEOUT(/Users/ordinateur/Projects/ansible/ansible.cfg) = 30
```

##### OS / ENVIRONMENT
- Target OS : Ubuntu 18.04.1
- Host OS: Mac OSX

##### STEPS TO REPRODUCE
```yaml
- name: Reboot a machine that might have lots of updates to apply
   reboot:
     reboot_timeout: 3600
```

##### EXPECTED RESULTS
It should reboot the machine and timeout only after 3600 s

##### ACTUAL RESULTS

```paste below
{"changed": false, "msg": "Invalid options for reboot: reboot_timeout"}
```

**Comment by ansibot** [comment]
Hi @Neophy7e, thank you for submitting this issue!

[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
Files identified in the description:
* [lib/ansible/modules/system/reboot.py](https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/system/reboot.py)

If these files are inaccurate, please update the `component name` section of the description or use the `!component` bot command.

[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by sivel** [comment]
Duplicate of https://github.com/ansible/ansible/pull/46585
